theSrc/scripts/LabeledScatter.js

```diff
@@ -54,7 +54,7 @@ class LabeledScatter {
   setConfig (data) {
     // Reset widget if previous data present but not equal in params - see VIS-278
     if (!(_.isUndefined(this.data)) && !(_.isUndefined(this.plot)) && !(this.plot.isEqual(data))) {
```

Comment:
I tested the branch on reusdev and it is working as per https://numbers.atlassian.net/browse/VIS-278

Approved.